validators/GST_IN.js

```diff
@@ -0,0 +1,21 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
+        return { success: false, message: 'GST number must be a Alphanumeric String!' }
```

Comment:
Alphanum check is also required along with `typeof` check

Reply:
the regex below will check that ,so its necessary over here imo